board/safety/safety_hyundai_canfd.h

```diff
@@ -369,14 +367,24 @@ static const addr_checks* hyundai_canfd_init(uint16_t param) {
   }
 
   if (hyundai_longitudinal) {
-    hyundai_canfd_rx_checks = (addr_checks){hyundai_canfd_long_addr_checks, HYUNDAI_CANFD_LONG_ADDR_CHECK_LEN};
+    if (hyundai_canfd_hda2) {
+      hyundai_canfd_rx_checks = (addr_checks){hyundai_canfd_hda2_long_addr_checks, ARRAY_LEN(hyundai_canfd_hda2_long_addr_checks)};
```

Comment:
make a macro like this?
```suggestion
      hyundai_canfd_rx_checks = SET_ADDR_CHECKS(hyundai_canfd_hda2_long_addr_checks);
```

Reply:
Done